tests/test_register_schema.py

```diff
@@ -0,0 +1,130 @@
+from contextlib import redirect_stderr
+import json
+import jsonschema
+import logging
+import pytest
+import io
+
+from jupyter_telemetry.eventlog import EventLog
+
+
+def test_register_invalid():
+    """
```

Comment:
I think it would be worth adding comments that describe the specific "failures" we are testing in each chunk below. 

Another approach would be to break each chunk into a separate unit test. I doesn't matter to me, but some of these failures aren't what I expect. See comments below.